optimism.tokenlist.json

```diff
@@ -914,6 +914,28 @@
       "decimals": 18,
       "logoURI": "https://ethereum-optimism.github.io/logos/WETH.png"
     },
+    {
+      "chainId": 1,
+      "address": "0x259ab9b9eab62b0fd98729b97be121073d5b3479",
```

Comment:
This is a minimal proxy EIP1167 contract is it? Is the implementation at `d1ac0b2a9be451e47c5a2bbab3dc22dc8abdd5dc`? If so can it be verified pls.

Reply:
Ah yep. it was verified through sourcify!
[0xD1Ac0B2a9BE451E47c5a2BBAB3DC22dC8aBDd5dc](https://repo.sourcify.dev/contracts/full_match/1/0xD1Ac0B2a9BE451E47c5a2BBAB3DC22dC8aBDd5dc/)